account_invoice_production_lot/models/invoice_line.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+# Copyright 2011 Domsense s.r.l. <http://www.domsense.com>
+# Copyright 2013 Lorenzo Battistini <lorenzo.battistini@agilebg.com>
+# Copyright 2017 Vicent Cubells <vicent.cubells@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class AccountInvoiceLine(models.Model):
+    _inherit = "account.invoice.line"
+
+    order_line_ids = fields.Many2many(
+        comodel_name='sale.order.line',
+        relation='sale_order_line_invoice_rel',
+        column1='invoice_id',
+        column2='order_line_id',
+        string='Order Lines',
+        readonly=True,
+        oldname='order_lines',
```

Comment:
As this is not a field with column, you don't need `oldname`. `relation`, `column1` and `column2` already point to the correct place to see for preserving data.